docs/getting-started/ios/fastlane-swift.md

```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminar support which allows the combination of other SPM packages with _fastlane_.
```

Comment:
This sentence needs clarification.

Reply:
Will do will do, my bad 🙇🏻‍♂️

Reply:
I totally thought `preliminar` was an English word 😂 two individuals that speak latin-like languages writing 🇪🇸  + reviewing 🇧🇷 the same code →  💥 😂😂